src/webapi/appsettings.json

```diff
@@ -21,5 +21,7 @@
     "NewUsersAreAdministrators": false,
     "RequireHttps": true
   },
-  "CorsAllowedOrigins": ""
+  "CorsAllowedOrigins": "",
+  "BasePath": "https://localhost:443/api/data/v3/",
+  "TrustAllSSLCerts": true
```

Comment:
The `appsettings.json` file should usually contain universal defaults. Both of the lines above are inappropriate for default values. We can override with _development machine defaults_ by adding them to `appsettings.Development.json`. For this file, we should simply have:

```json
  "BasePath": ",
  "TrustAllSSLCerts": false
```